src/app/directives/response-short.js

```diff
@@ -0,0 +1,20 @@
+(function () {
+  'use strict';
+
+  RAML.Directives.responseShort = function() {
+    return {
+      restrict: 'E',
+      templateUrl: 'directives/response-short.tpl.html',
+      scope: {
+        code: '=',
+        info: '='
+      },
+      controller: ['$scope', function($scope) {
+        $scope.description = RAML.Transformer.transformValue($scope.info.description());
+      }]
+    };
+  };
+
+  angular.module('RAML.Directives')
```

Comment:
All components in this project follow this convention. A change of that nature will need a separate PR, as doing otherwise will leave things inconsistent.